tests/test_windows.rs

```diff
@@ -89,6 +89,7 @@ fn canonicalize_root() {
     // TODO:
     // This works here, but it doesn't work when called in src/arc.rs::handle_prefix
     // during `test_absolute`, possibly related to setting the current_dir?
+    // TODO: Doesn't work when run from a drive other than C:.
```

Comment:
I think this test can be removed now?

Reply:
I didn't want to delete any of your tests uninvited, but yes, I think this test is redundant with `absolute_supports_root_only_relative_path()`, so I've removed it.